Peer dependency update to react 17

Hey, I'm using react-tiny-popover with React 17 and it all works fine.

It'd be great to update the peer dependencies to support React 17 so that installing the package doesn't result in NPM peer dependency errors 😄 

```bash
$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: app@1.0.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0.0" from react-tiny-popover@6.0.4
npm ERR! node_modules/react-tiny-popover
npm ERR!   react-tiny-popover@"^6.0.4" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/place1/.npm/eresolve-report.txt for a full report.
p
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/place1/.npm/_logs/2021-02-01T02_48_05_526Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-tiny-popover",
-  "version": "6.0.5",
+  "version": "6.0.6",
   "description": "A simple and highly customizable popover react higher order component with no other dependencies! Typescript friendly.",
   "keywords": [
     "react",
@@ -52,7 +52,7 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^16.0.0 || ^17.0.0",
+    "react-dom": "^16.0.0 || ^17.0.0"
   }
 }

